### DISCUSSION TITLE
Compare with Nemos' MarbleNet

### DISCUSSION THREAD

**Discussion by MonolithFoundation** [snakers4/silero-vad#593] [Q&A]
MarbleNet was introduced in 2020. Nevertheless, it still yields better results than many open-source VAD models. Is there any consideration for making a comparison with it? In my situation, MarbleNet is superior to silero. However, MarbleNet is extremely difficult to use due to its deep coupling with NEMO and is not as clean as silero.

**Comment by snakers4** [comment]
Silero is a streaming VAD with a 30-ms chunk. If nothing changed since I last checked, MarbleNet is a synchronous VAD with a ~1s chunk. Please correct me, if I am wrong.

Also it was orders of magnitude slower.

**Reply by MonolithFoundation** [reply]
“Hello, I am quite new to this field. Which one might be better? A 1-second chunk or a 30-millisecond chunk? I have been using SilveR all the time, and it actually functions well, except that some human speech might be missed. I am not sure if it is due to the chunk size.”
